docs/datafeed.md

```diff
@@ -0,0 +1,136 @@
+# Datafeed
+
+The Datafeed Service is a service used for handling the [_Real Time Events_](https://developers.symphony.com/restapi/docs/real-time-events). 
+For each interaction of a user within the IM, MIM or Room chat, an event will be sent to the datafeed.
+The bot can create a datafeed, list all created datafeeds or retrieve all the Real Time events within a datafeed through Datafeed API.
+The Datafeed Service is a Core service built on top of the Datafeed API and provide a dedicated contract to bot developers to work with datafeed. 
+
+## How to use
+The central component for the contract between bot developers and  the Datafeed API is the `DatafeedService`.
+This service is accessible from the `SymphonyBdk` object by calling the `datafeed()` method.
+For instance:
+
+```java
+public class Example {
+
+    public static void main(String[] args) {
+
+        final SymphonyBdk bdk = new SymphonyBdk(loadFromClasspath("/config.yaml"));
+
+        bdk.datafeed().subscribe(new RealTimeEventListener() {                                 // (2)
+    
+            @Override
+            public void onMessageSent(V4Initiator initiator, V4MessageSent event) {
+                bdk.messages().send(event.getMessage().getStream(), "<messageML>Hello, World!</messageML>");
+            }
+        });
+
+        bdk.datafeed().start();  
+    }
+}
+```
+
+A more detailed example of the usage of the Datafeed service can be found [here](../symphony-bdk-examples/bdk-core-examples/src/main/java/com/symphony/bdk/examples/DatafeedExampleMain.java).
+
+## Datafeed Configuration
+
+Datafeed Service can be configured by the datafeed field in the configuration file:
+
+```yaml
+datafeed:
+  version: 'v1' # specify datafeed version 'v1' or 'v2'
+  retry:
+    maxAttempts: 6 # maximum number of retry attempts
+    initialIntervalMillis: 2000 # initial interval between two attempts
+    multiplier: 1.5 # interval multiplier after each attempt
+    maxIntervalMillis: 10000 # limit of the interval between two attempts
+```
+
+The minimal configuration for the datafeed service is the version of the datafeed which will be chosen to be use in the bdk.
+If the bot developers want to use the datafeed version 2 in their bot, the version configuration have to be specified as `v2`.
```

Comment:
we should probably explain how as a bot developer you can find out wether to use v1 or v2